Libplanet/Blocks/Block.cs

```diff
@@ -513,13 +526,9 @@ public byte[] Serialize()
         /// <exception cref="InvalidBlockNonceException">Thrown when
         /// the <see cref="Nonce"/> does not satisfy its
         /// <see cref="Difficulty"/> level.</exception>
-        /// <exception cref="InvalidTxSignatureException">Thrown when its
-        /// <see cref="Transaction{T}.Signature"/> is invalid or not signed by
-        /// the account who corresponds to its
-        /// <see cref="Transaction{T}.PublicKey"/>.</exception>
-        /// <exception cref="InvalidTxPublicKeyException">Thrown when its
-        /// <see cref="Transaction{T}.Signer"/> is not derived from its
-        /// <see cref="Transaction{T}.PublicKey"/>.</exception>
```

Comment:
One thing I wonder about this: Are these exception classes still used from any other places? Can't they be safely removed now?

Reply:
They're still thrown when staging tx on `BlockChain<T>` or calling `Transaction<T>.Validate()` manually.